### PR TITLE
Add DB Config

### PR DESCRIPTION
**Pull request by anmoldtu**
Add Postgres Setup for running unit tests on Github CICD

### CODE REVIEW

**Comment by Yamini-L** [comment]
Looks good!

**Review by DaveCSW — APPROVED** [review]
good job

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -54,9 +54,41 @@ jobs:
...
+            timeout=$((timeout-5))
+            if [ $timeout -le 0 ]; then
+              echo "Timed out waiting for PostgreSQL to be ready."
+              exit 1
+            fi
+          done
+      
+    - name: Load SQL Data
+      run: PGPASSWORD=123456 psql -U $POSTGRES_USER -d $POSTGRES_DB -W -h localhost -p 5432 -f ../sql/create_tables.sql
+
```
Comment (gurnoor6) [inline .github/workflows/run-tests.yml:91]: Good job! I think let us find a way to hide this detail from the public. Even though the password here is not sensitive information, I still think we should encapsulate this information.

**Review by Yamini-L — APPROVED** [review]
Looks good!

File: backend/events/tests/.env [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# PostgreSQL Configuration
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=take-a-break
+DB_USER=postgres
+DB_PASSWORD=123456
```
Comment (gurnoor6) [inline backend/events/tests/.env:6]: Let us add `.env.example` file with this data

**Review by gurnoor6 — APPROVED** [review]
Good job! Approving for now. Created an issue #99 for the changes.

**Pull request merged**